kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/KafkaProxyReconciler.java

```diff
@@ -83,22 +84,24 @@ public class KafkaProxyReconciler implements
     public static final String CONFIG_DEP = "config";
     public static final String DEPLOYMENT_DEP = "deployment";
     public static final String CLUSTERS_DEP = "clusters";
-    static final String SEC = "sec";
+
+    private final Clock clock;
     private final SecureConfigInterpolator secureConfigInterpolator;
 
-    public KafkaProxyReconciler(SecureConfigInterpolator secureConfigInterpolator) {
+    public KafkaProxyReconciler(Clock clock, SecureConfigInterpolator secureConfigInterpolator) {
+        this.clock = clock;
         this.secureConfigInterpolator = secureConfigInterpolator;
     }
 
     static SecureConfigInterpolator secureConfigInterpolator(Context<KafkaProxy> context) {
-        return context.managedWorkflowAndDependentResourceContext().getMandatory(SEC, SecureConfigInterpolator.class);
+        return KafkaProxyContext.secureConfigInterpolator(context);
     }
 
     @Override
     public void initContext(
-                            KafkaProxy primary,
+                            KafkaProxy proxy,
```

Comment:
Generally speaking using `primary` is now confusing, because what the primary is depends on the reconciler or dependent resource. Better to be explicit, I think.

Reply:
yep think it's a win, primary isn't helping